R/RtGam.R

```diff
@@ -0,0 +1,28 @@
+#' Stub: Will become fitting/class constructor
+#'
+#' Including as a stub function to showcase control flow
+#' @param cases A vector of non-negative incident case counts occurring on an
+#'   associated `reference_date`. Missing values (NAs) are not allowed.
+#' @param reference_date The associated date on which the count of incident
+#'   `cases` occurred. Missing dates are not allowed and dates can only occur
+#'   once.
+#' @param group The geographic grouping for the case/reference-date pair. Not
```

Comment:
From this I take it you aren't planning on expose the `mgcv` formula interface?

Reply:
Yeah not currently planning on exposing it. Maybe would revisit post v0.1.0?